Extensions/quick_tags.js

```diff
@@ -49,7 +54,8 @@ XKit.extensions.quick_tags = new Object({
 	menu_close: function() {
 		// Only close the menu if it doesn't have keyboard or mouse focus
 		if ($("#xkit-quick-tags-window").find('input:focus').length === 0 &&
-		    $('#xkit-quick-tags-window:hover').length === 0) {
+		    $('#xkit-quick-tags-window:hover').length === 0 &&
+		    Xkit.extensions.quick_tags.preferences.close_on_focus.value === false) {
```

Comment:
Try using `!Xkit.extensions.quick_tags.preferences.close_on_focus.value` for two reasons:

1. `Xkit` isn't defined, but `XKit` is
2. In general, if statements that are like `if (something === true)` or `if (something === false)` are redundant; instead, we can use `if (something)` or `if (!something)`